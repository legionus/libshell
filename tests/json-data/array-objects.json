[{"foo0":"bar0"},{"foo1":"bar1"},{"foo2":"bar2"}]
