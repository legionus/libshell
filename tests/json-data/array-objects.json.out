/0/foo0 string bar0
/1/foo1 string bar1
/2/foo2 string bar2
